certified-connectors/emailDomainChecker/Readme.md

```diff
@@ -0,0 +1,51 @@
+# Email Domain Checker 
+

```

Comment:
Hello @mightora, 

Can you please update readme.md file as per the template in the below url?
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
@vmanoharas  I have updated the file